src/components/content-tab-snapshots.tsx

```diff
@@ -410,6 +430,7 @@ function AddDemoSiteWithProgress( {
 		snapshotQuota
 	);
 	const offlineMessage = __( 'Creating a demo site requires an internet connection.' );
+	const userBlockedMessage = __( 'Demo sites are not available for your account.' );
```

Comment:
I wonder if we could reuse the string from `useArchiveErrorMessages` hook:
https://github.com/Automattic/studio/blob/5b834ea553f5097dc9caeee0fb6baecb4994ba52/src/hooks/use-archive-error-messages.ts#L23

Reply:
I fixed that in https://github.com/Automattic/studio/pull/354/commits/0b95433de9d3b9d7107a3225448ac7d57a4aea4f